### PR TITLE
x11-plugins/pidgin-funyahoo-plusplus: New package

### PR DESCRIPTION
**Pull request by ljakab**
Gentoo-bug: 591650

### CODE REVIEW

**Comment by gktrk** [comment]
@gentoo/proxy-maint

File: x11-plugins/pidgin-funyahoo-plusplus/Manifest [hunk 1]
```diff
@@ -0,0 +1 @@
+EBUILD pidgin-funyahoo-plusplus-9999.ebuild 422 SHA256 30d959c27e2eb20245cee5c08ce2017d3cad5ba971a705e9ae11078770bd9463 SHA512 20866b89c93e656eb0ae00b9a928fb44b1dace17b72f4195e44910f9bb31c16ecb5cf3b45d72c8a4cfd90642a1b96e5e0a182d518b5fb2634b21570b377efaec WHIRLPOOL 5b7e5cb16b025065e63cc3fd3d5366418e48ee91368325f4f9da812ffdd295a54beb212b996d97d8df2d70c4ce6706281c36613322d8bfff27cb2c53a29ac3a1
```
Comment (Coacher) [inline x11-plugins/pidgin-funyahoo-plusplus/Manifest]: Please use thin Manifests: https://wiki.gentoo.org/wiki/Repository_format/package/Manifest#Thin_Manifest

File: x11-plugins/pidgin-funyahoo-plusplus/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<!-- maintainer-needed -->
```
Comment (Coacher) [inline x11-plugins/pidgin-funyahoo-plusplus/metadata.xml]: Please don't add new packages with no maintainers.
See https://bugs.gentoo.org/show_bug.cgi?id=590972 and https://projects.gentoo.org/council/meeting-logs/20160814-summary.txt
If you plan to maintain this package via proxy-maint like you've said in the bug, please reflect this fact in metadata.

**Comment by ljakab** [comment]
Updated pull request based on you comments, many thanks for the help!

**Pull request closed without merging**